### ISSUE TITLE
icon-request fa-cookies

### ISSUE THREAD

**Issue by tverheijen** [FortAwesome/Font-Awesome#8767]
fa-cookies would be nice

**Comment by tagliala** [comment]
Duplicate of #2845 ?

**Comment by tverheijen** [comment]
Yes sorry, still waiting on that cookie fa :)

**Comment by CharlSteynberg** [comment]
Have a look here:
https://github.com/CharlSteynberg/glyphs/tree/master/FontAwesome
